PlanGrid.Api/Project.cs

```diff
@@ -10,5 +10,41 @@ public class Project : Record
     {
         [JsonProperty("name")]
         public string Name { get; set; }
+
+        [JsonProperty("custom_id")]
+        public string CustomId { get; set; }
+
+        [JsonProperty("type")]
+        public string Type { get; set; }
+
+        [JsonProperty("status")]
+        public string Status { get; set; }
+
+        [JsonProperty("owner")]
+        public string Owner { get; set; }
+
+        [JsonProperty("start_date")]
+        public DateTime? StartDate { get; set; }
```

Comment:
`StartDate` and `EndDate` should be `Date?`s for both classes as they're ISO dates and are nullable. Check out the [project object](https://developer.plangrid.com/docs/the-project-object) to see their format.